XVim/Test/XVimTestCase.m

```diff
@@ -68,6 +69,7 @@ - (void)setUp{
     [[[XVimLastActiveSourceView() xvim_window] sourceView] xvim_changeSelectionMode:XVIM_VISUAL_NONE];
     [XVimLastActiveSourceView() setString:self.initialText];
     [XVimLastActiveSourceView() setSelectedRange:self.initialRange];
+    [XVim instance].options.expandtab = YES;
```

Comment:
Do you need this here? This setup method is commonly used by all the test cases. I don't want to put test case specific setup here.